src/input/mpegts/iptv/iptv.c

```diff
@@ -614,14 +614,14 @@ const idclass_t iptv_network_class = {
     {
       .type     = PT_U32,
       .id       = "max_streams",
-      .name     = N_("Maximum Input Streams"),
+      .name     = N_("Maximum input streams"),
```

Comment:
Should probably be "maximum number of input streams" or even "Maximum # input streams" in US vernacular